test/rubocop-diff-test

```diff
@@ -24,18 +24,12 @@ function assert_equals() {
   fi
 }
 
-RED='\033[0;31m'
-GREEN='\033[0;32m'
-NC='\033[0m'
-
 current_path=$(pwd)
-path=~/tmp/rubocop-diff
-rm -rf "$path"
-mkdir -p "$path"
+path=$(mktemp -d ~/rubocop-diff.XXXXXXXXXX)
```

Comment:
Still creating directories in the user's home directory, this will lose you friends.  Why not use the `-t` option?
```
>  mktemp -d 'rubocop-XXXXXX' -t
/tmp/rubocop-7YZg64
```
or at least make the directory configurable with `-p`
```
>  mktemp -d 'rubocop-XXXXXX' -p /tmp
/tmp/rubocop-7YZg64
```